components/GardenPage/GardenPage.js

```diff
@@ -0,0 +1,379 @@
+import React, { useState } from "react";
+import styled, { keyframes, css } from "styled-components";
+import Link from "next/link";
+//import petsData from "@/lib/Data";
+
+const rotate = keyframes`
+  from {
+    transform: rotate(0deg);
+  }
+  to {
+    transform: rotate(360deg);
+  }
+`;
+
+const bounce = keyframes`
+  0%, 35%, 55%, 80%, 100% {
+    transform: translateY(0);
+  }
+  5% {
+    transform: translateY(-45px);
+  }
+  40% {
+    transform: translateY(-30px);
+  }
+  60% {
+    transform: translateY(-15px);
+  }
+`;
+
+const grow = keyframes`
+  0%, 100% {
+    transform: scale(1);
+  }
+  50% {
+    transform: scale(1.2);
+  }
+`;
+
+const GardenContainer = styled.div`
+  position: relative;
+  width: 100%;
+  height: 90vh;
+  background-image: url("/Background/pet-app-background-dalle.jpg");
+  background-size: cover;
+  background-position: center;
+  background-repeat: no-repeat;
+`;
+
+const PetWrapper = styled.div`
+  position: absolute;
+  left: 50%;
+  top: 50%;
+  transform: translate(-50%, -50%);
+`;
+
+const PetDisplay = styled.div`
+  font-size: 8em;
+  color: var(--text-color);
+  transform-origin: center;
+  animation: ${({ $rotating, $bouncing, $growing }) =>
+    $rotating
+      ? css`
+          ${rotate} 1s linear
+        `
+      : $bouncing
+      ? css`
+          ${bounce} 1s ease
+        `
+      : $growing
+      ? css`
+          ${grow} 0.5s ease
+        `
+      : "none"};
+`;
+
+const NavbarContainer = styled.div`
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 5px;
+  background-color: var(--neutral-color);
+  border-radius: 8px;
+  margin: auto;
+  max-width: 600px;
+`;
+
+const NavButton = styled.button`
+  background-color: var(--primary-color);
+  color: var(--text-color);
+  border: none;
+  padding: 10px;
+  text-align: center;
+  text-decoration: none;
+  display: inline-block;
+  font-size: 16px;
+  margin: 4px;
+  cursor: pointer;
```

Comment:
I am unsure if there is a differing definition in the rest of the code but on hovering the aoory does not transfomr into a pointer

Reply:
Links and other interactive elements, like buttons, have cursor: pointer; applied by default in most browsers.

The issue arises because the clickable link is placed inside a `div`. This is another reason why it’s better to move the link out of the `div`.